src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,16 +1,70 @@
-import { Component } from 'react';
+import React, { useState } from 'react';
 
-type Props = {};
-type State = {};
+interface Props {
+  onAdd: (movie: Movie) => void
+}
 
-export class NewMovie extends Component<Props, State> {
-  state: State = {};
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
 
-  render() {
-    return (
-      <form>
-        Put the form here
-      </form>
-    );
-  }
-}
+  const resetForm = () => {
+    setTitle('');
+    setDescription('');
+    setImgUrl('');
+    setImdbUrl('');
+    setImdbId('');
+  };
+
+  const createMovie = () => {
+    onAdd({
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+    });
+  };
+
+  const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    createMovie();
+    resetForm();
+  };
+
+  return (
+    <form onSubmit={onSubmit}>
+      <input
+        placeholder="Enter title"
+        value={title}
+        onChange={(e) => setTitle(e.target.value)}
```

Comment:
Не скорочуй назви змінних
```suggestion
        onChange={(event) => setTitle(e.target.value)}
```